server/config/routes.js

```diff
@@ -207,8 +207,12 @@ exports.configure = function configure(app) {
   app.get('/ledgers/debtor_sale/:id/:saleId', ledger.compileDebtorLedgerSale);
 
   /* fiscal year controller */
-  app.get('/fiscal', fiscal.getFiscalYears);
-  app.post('/fiscal/create', fiscal.createFiscalYear);
+  app.get('/fiscal', fiscal.list);
+  app.get('/fiscal/date', fiscal.getFiscalYearsByDate);
+  app.get('/fiscal/:id', fiscal.detail);
+  app.post('/fiscal/create', fiscal.create); 
```

Comment:
These URLs do not need to match what is done on the client.  Please stick to the API designs specified in [this wiki article](https://github.com/IMA-WorldHealth/bhima-2.X/wiki/Design:-Server-CRUD-APIs).